android/app/build.gradle

```diff
@@ -30,7 +30,8 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 34
+    namespace 'ch.nevis.nevis_mobile_authentication_sdk_example_app_flutter'
+    compileSdk 34
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_11
         targetCompatibility JavaVersion.VERSION_11
```

Comment:
🔧 I think this, and also the `jvmTarget` under `kotlinOptions` should be set to `17` as well. I'm actually surprised the project compiles this way, for me (when I tried with a native Android project, I think the _Access App_) it refused to compile if Kotlin were used with a newer Gradle.

Reply:
Fixed.